enigma-tools-u/src/common_u/logging.rs

```diff
@@ -73,3 +86,25 @@ pub fn init_logger<P: AsRef<Path>>(level: log::LevelFilter, data_dir: P, name: S
     let handle = log4rs::init_config(config).unwrap();
     Ok(handle)
 }
+
+
+#[cfg(test)]
```

Comment:
As we discussed, you may remove this test since it does not test our functionality.